pkg/kubectl/cmd/apply_test.go

```diff
@@ -403,3 +405,87 @@ func testApplyMultipleObjects(t *testing.T, asList bool) {
 		t.Fatalf("unexpected output: %s\nexpected: %s OR %s", buf.String(), expectOne, expectTwo)
 	}
 }
+
+const (
+	filenameDeployObjServerside = "../../../test/fixtures/pkg/kubectl/cmd/apply/deploy-serverside.yaml"
+	filenameDeployObjClientside = "../../../test/fixtures/pkg/kubectl/cmd/apply/deploy-clientside.yaml"
+)
+
+func validateNULLPatchApplication(t *testing.T, req *http.Request) {
```

Comment:
Sorry I'm not sure I follow. Isn't this what the other tests in apply_test.go are doing? Non of the other tests have a defined null value in their applied object.